WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -209,6 +209,11 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             case RequestCodes.CREATE_SITE:
                 if (resultCode == RESULT_OK) {
                     debounceLoadSites();
+
+                    if (data != null) {
```

Comment:
If `data` is null should we create it here? Looking at `SiteCreationActivity` I see [one situation](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/accounts/SiteCreationActivity.java#L148) where the result with be `RESULT_OK` and `data` will be null even if a site has been created.